hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/TimelineServerBasedWriteMarkers.java

```diff
@@ -132,6 +153,25 @@ public Set<String> allMarkerFilePaths() {
     }
   }
 
+  @Override
+  public void createMarkerDir() throws HoodieIOException {
+    HoodieTimer timer = new HoodieTimer().startTimer();
+    Map<String, String> paramsMap = new HashMap<>();
+    paramsMap.put(MARKER_DIR_PATH_PARAM, markerDirPath.toString());
```

Comment:
Do we need a HTTP request for creating the marker dir then, the server already located on the driver, and we already create a marker dir when start a new instant.

Reply:
Currently, the Timeline server based markers is designed using this mechanism.  Mainly used for cloud based solutions. @nsivabalan @yihua  can add details.

Reply:
I was thinking whether it is necessary to create/drop a dir with additional HTTP overhead, I saw we already did that for drop marker dir, so I'm fine with it.